core/src/serve_repair.rs

```diff
@@ -746,7 +723,6 @@ impl ServeRepair {
         stats: &mut ServeRepairStats,
         data_budget: &DataBudget,
     ) {
-        let sign_repairs_epoch = Self::sign_repair_requests_activated_epoch(root_bank);
         let check_ping_ancestor_request_epoch =
             Self::check_ping_ancestor_requests_activated_epoch(root_bank);
```

Comment:
need to get rid of this one as well. the feature for ancestor hashes

Reply:
removed